hbase-server/src/test/java/org/apache/hadoop/hbase/io/hfile/TestLruBlockCache.java

```diff
@@ -141,7 +141,7 @@ public void testBackgroundEvictionThread() throws Exception {
     }
 
     // wait until at least one eviction has run
-    Waiter.waitFor(CONF, 30000, 200, new ExplainingPredicate<Exception>() {
+    Waiter.waitFor(CONF, 31000, 200, new ExplainingPredicate<Exception>() {
```

Comment:
This fix is entirely dependent on the runtime environment. Can we instead monitor some event or side effect to improve test robustness?